lighthouse-cli/test/smokehouse/smokehouse-bin.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const path = require('path');
+const yargs = require('yargs');
+const log = require('lighthouse-logger');
+
+const {runSmokehouse} = require('./smokehouse.js');
+const {server, serverForOffline} = require('../fixtures/static-server.js');
+
+const cliLighthouseRunner = require('./run-lighthouse-cli.js').runLighthouse;
+
+const coreTestDefnsPath = require.resolve('./test-definitions/core-tests.js');
+
+const runners = {
+  cli: cliLighthouseRunner,
+};
+
+/* eslint-disable no-console */
+
+/**
+ * @fileoverview Run smokehouse from the command line. Run webservers,
```

Comment:
should this be higher?

Reply:
> should this be higher?

probably. I'll also add more in more files